pom.xml

```diff
@@ -2508,6 +2584,25 @@
         <orc.flink.version>1.5.6</orc.flink.version>
         <flink.avro.version>1.11.1</flink.avro.version>
       </properties>
+      <build>
```

Comment:
Flink from 1.17 to 1.13 all depends on 1.10.1 parquet version, which does not support index metadata, like column index. We have to exclude these classes from these profiles.